components/eamxx/src/share/io/scream_io_utils.hpp

```diff
@@ -80,9 +80,9 @@ struct LongNames {
   std::map<std::string,std::string> name_2_longname = {
```

Comment:
Would it make sense to do away with this approach to long names altogether and just use the same syntax for `hya`, `hyb`, `lev`, etc. as we do for the diagnostics?  I.e. add longname as metadata?

Reply:
Probably. But we can do it in a separate PR.